css/style.css

```
@@ -0,0 +1,151 @@
+:root {
+    --main-color: #36281d;
+    --news-font: 'Montserrat', sans-serif;
+}
+
+body {
+    font-family: var(--news-font);
+    font-weight: 200;
+    font-style: italic;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+}
+
+.A4 {
+    width: 21cm;
+    height: 29.7cm;
+    margin: auto;
+    padding: 1cm;
+    display: flex;
+    flex-direction: column;
+    background-image: url('/images/old-paper.jpg');
```

Картинка здесь просто гигантская, из-за этого при загрузке виден контент без неё. Хорошее решение, если совсем никак нельзя будет отказать от неё, задать цвет по умолчанию

---

добавил цвет по умолчанию
```suggestion
    background-color: blanchedalmond;
    background-image: url('/images/old-paper.jpg');
```